flask_resty/pagination.py

```diff
@@ -542,10 +560,28 @@ def get_filter_clause(self, column_cursors):
             value = int(value)
 
         if asc:
-            current_clause = column > value
+            if value is None:
+                return None
+            elif value is not None:
+                current_clause = self._handle_nullable(
+                    column, value, is_nullable
+                )
```

Comment:
the assumption here is that the column is always `nulls first` for ascending, right?

Reply:
sorry, ok I misspoke asc -> nulls last. So for non NULL values we have to assume there may be upcoming null values, hence the handling